static/app/controllers.js

```diff
@@ -11,23 +11,15 @@
             .then(renderStatus);
     });
 
-    app.controller('DashboardCtrl', function($scope, $cloudfoundry, $location) {
+    app.controller('MainCtrl', function($scope, $cloudfoundry, $location, $rootScope) {
```

Comment:
So originally, I depended on root scope for the active org but it made bookmarking/refreshing a problem. For example, if you refresh on a space page the activeOrg will stay undefined because the activeOrg is set with the org controller. Hence, the findActiveOrg method... There might be a better way though.